### ISSUE TITLE
Interested in collaboration?

### ISSUE THREAD

**Issue by birdayz** [Hosq/BigWigs#11]
Hi there,
i've also been working on a ported bigwigs for elysium in the last weeks: https://github.com/birdayz/BigWigs
Beside some exceptions, timers are very accurate. I took timers from Elysium's server code and test it on my own installation of the elysium core ( i guess you do this as well)

However, i'm using the "old" BigWigs from LYQ as base, which was - which i now know - a bad choice. It is (in my opinion) not very good from a code quality point of view (no offense to anyone :) ) - tons of copy/pasted code, magic variables everywhere (however, timers for mc/bwl content are very accurate).
I recently came across your repo, which uses xorann's BigWigs as base, which includes much required refactoring to the code to become maintainable. I was also considering porting my stuff to xorann's Bigwigs, but i think we could just join our efforts instead of working in parallel on our own stuff :)

TL;DR i would like to contribute some stuff to your repo, since you're on a much better codebase anyway! I got some more timers, like knock away on broodlord etc, and some timers are maybe a little more accurate. I'd like to have a single BigWigs for Elysium, which is near-100 % :)

Edit: nvm you have the knock away, bad example :)

**Comment by Hosq** [comment]
Yea working together sounds good. Add me on discord if you wanna talk more Hosq#0708

**Comment by 0ldi** [comment]
Hosq version is pretty accurate in itself, I think if you offer specific fixes (and they will be correct), it will be fine.
